### PR TITLE
Quotes in the cookies are not escaped when a task script for an iframe is built on the client (close #366)

### LINKED ISSUE #366: Quotes in the cookies are not escaped when a task script for an iframe is built on the client
http://66.55.147.18:13500/history?timestamp=1453358894078&siteUrl=http://www.flipkart.com/

### PR DESCRIPTION
**Pull request by churkin**
\сс @LavrovArtem @miherlosev

### CODE REVIEW

**Comment by inikulin** [comment]
Regression tests?

**Comment by churkin** [comment]
FPR

File: test/client/fixtures/sandbox/iframe-test.js [hunk 1]
```diff
@@ -201,6 +202,23 @@ asyncTest('native methods are properly initialized in an iframe without src (GH-
     document.body.appendChild(iframe);
 });
 
+asyncTest('quotes in the cookies are not escaped when a task script for an iframe is built on the client (GH-366)', function () {
+    var iframe = document.createElement('iframe');
+    var cookie = '""\'\'';
+
+    iframe.id             = 'test_cookie';
+    settings.get().cookie = cookie;
+
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/iframe-test.js]: Before `start` calling we need to restore origin `settings.get().cookie` value.

**Comment by miherlosev** [comment]
\r-

File: test/client/fixtures/sandbox/iframe-test.js (hunk 1 above)
Comment (churkin) [inline test/client/fixtures/sandbox/iframe-test.js]: We do not have default cookie value, so I think it is not necessary

Comment (miherlosev) [inline test/client/fixtures/sandbox/iframe-test.js]: So, ok

**Comment by LavrovArtem** [comment]
lgtm

**Pull request merged**